odoo/addons/base/models/res_users.py

```diff
@@ -1921,6 +1929,24 @@ def change_password(self):
         # would be great to update the session id in-place, but it seems dicey
         return {'type': 'ir.actions.client', 'tag': 'reload'}
 
+class RevokeAllDevices(models.TransientModel):
+    _name = "revoke.all.devices"
+    _description = "User, revoke all devices wizard"
+    _transient_max_hours = 0.1
```

Comment:
that's 6min, maybe it's a bit shot for the "hmm where did I put my password again ? was it on this sheet of paper.. oh no wait" kind of scenario.